apps/frontend/src/routes/register/verify/index.tsx

```diff
@@ -35,6 +41,22 @@ export default component$(() => {
     });
   });
 
+  useTask$(({ track }) => {
+    const resent = track(() => store.resent);
+    if (!resent) return;
```

Comment:
To make it more readable please write the return in a block.

Reply:
Done